includes/RequestWiki/SpecialRequestWiki.php

```diff
@@ -111,17 +118,18 @@ public function onSubmit( array $formData ) {
 
 		$out = $this->getOutput();
 
-		// Make the subdomain a dbname
-		if ( !ctype_alnum( $subdomain ) ) {
+		if ( $wmError ) {
+			$out->addHTML( '<div class="errorbox">' .  $this->msg( 'createwiki-error-dbexists' )->escaped() . '</div>' );
```

Comment:
You may want to use ``Html::errorBox`` here because of the 101 class name changes they are working on upstream. I think ``errorbox`` will soon be renamed, too.

Reply:
I've been planning to do this in mass on a future PR.

Reply:
Marking this review as resolved, since I'll fix in a future PR.